docs/src/main/asciidoc/_configprops.adoc

```diff
@@ -35,6 +35,7 @@
 |spring.cloud.loadbalancer.health-check.interval | 25s | Interval for rerunning the HealthCheck scheduler.
 |spring.cloud.loadbalancer.health-check.path |  | 
 |spring.cloud.loadbalancer.hint |  | Allows setting the value of <code>hint</code> that is passed on to the LoadBalancer request and can subsequently be used in {@link ReactiveLoadBalancer} implementations.
+|spring.cloud.loadbalancer.retry.avoid-previous-instance | true | Enables wrapping ServiceInstanceListSupplier beans with `RetryAwareServiceInstanceListSupplier` if Spring-Retry is in the classpath.
```

Comment:
Add back ticks around ServiceInstanceListSupplier

Also, all the values that are code should be wrapped in back ticks. That's true in this line, and I see code values in some of the other lines, too.

Reply:
This file is automatically generated.